build.gradle

```diff
@@ -16,7 +16,7 @@ subprojects {
         implementation "com.google.guava:guava:29.0-jre"
         implementation "org.apache.logging.log4j:log4j-core:2.11.1"
         implementation "org.slf4j:slf4j-api:1.7.30"
-        implementation "org.slf4j:slf4j-log4j12:1.7.30"
+        implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.14.0"
```

Comment:
Nit: The "2.14.0" version is re-used in a few places. Seems that it can be added to the version map

Reply:
Removed most of the version map in #347 in favor of automated dependency management (just FYI)